index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+  
+
+
+	<div class="header-background" >
+		<div class="nav">
+		<a href="" class="m-logo" >My Logo</a>
+
+		<ul>
+         <li class="link-right">
+              
+              <a href="/someurl" class="inline"><b><img src="images/menu.PNG" alt="ss"></b></a>
+          </li>
+         </ul>
+     </div>
```

Comment:
- [ ] Great job 👍 . Kindly make sure you use a `header` tag to wrap around your `logo` and `menu` icon instead of a `div` tag 🙏 . This would make your code more semantic and very easy for people who rely on screen readers to navigate through. You can read about semantic HTML from this [article](https://www.w3schools.com/html/html5_semantic_elements.asp)